helmfile/values/userCRDs/userCRDs.yaml.gotmpl

```diff
@@ -1,8 +1,12 @@
 userCRDs:
     enabled: {{ .Values.gatekeeper.allowUserCRDs.enabled }}
     admin:
-        users: {{ .Values.clusterAdmin.users }}
-        groups: {{ .Values.clusterAdmin.groups }}
+        users:
+        - kubernetes-admin
+        {{- with .Values.clusterAdmin.users }}
+        {{- toYaml . | nindent 8 }}
+        {{- end }}
+        groups: {{- toYaml .Values.clusterAdmin.groups | nindent 10 }}
```

Comment:
```suggestion
        groups: {{- toYaml .Values.clusterAdmin.groups | nindent 8 }}
```

Reply:
It needs to be set to 10 here otherwise it breaks when `groups=[]`:
```console
        groups:
        []
  ...
COMBINED OUTPUT:
  ...
  Error: failed to parse /tmp/helmfile3174189742/gatekeeper-system-gatekeeper-constraints-values-6c489bd5c8: error converting YAML to JSON: yaml: line 12: could not find expected ':'
  Error: plugin "diff" exited with erro
```

I can set `nindent` to 10 for both `users` and `groups` if you prefer, but then `- kubernetes-admin` needs to be indented as well, otherwise all items in the `users` list will be added as one list item only. With how it is configured currently in this PR, if `users` is set to an empty list it is not added since it is enclosed with an `with` clause.

Reply:
Then go for 10 on both

Reply:
I noticed when I changed the indentation that setting `nindent` to 10 looked a bit weird as the rest of the file used 4 spaces for indentation, so I changed the file to use 2 spaces instead as that seemed to be more frequently used in other templating files found in the repo. It would be nice to maybe enforce some kind of code standard for yaml/templating files :sweat_smile:

Reply:
I just learned that we have this file `https://github.com/elastisys/compliantkubernetes-apps/blob/main/.editorconfig` that should enforce it, if you use editorconf plugin. I just installed it :slightly_smiling_face: